### PR TITLE
Update install instructions for Fedora 39 CSB

### PR DESCRIPTION
**Pull request by aireilly**
https://redhat-documentation-vale-at-red-hat-706.surge.sh/docs/main/user-guide/installing-vale-cli/

### CODE REVIEW

**Review by rolfedh — APPROVED** [review]
Excellent update! Thanks, Aidan. LGTM

**Comment by daobrien** [comment]
mczernek/vale is no longer maintained ?
 
 Are you referring to the COPR repo? It's still maintained, it just doesn't have the latest build right now.

**Comment by aireilly** [comment]
> mczernek/vale is no longer maintained ?
> 
> Are you referring to the COPR repo? It's still maintained, it just doesn't have the latest build right now.

Apologies. Let me rephrase.

File: modules/user-guide/pages/installing-vale-cli.adoc [hunk 1]
```diff
@@ -19,13 +19,35 @@ To use the various Vale at Red Hat styles, use Vale version 2.30.0 or lower.
...
-$ sudo dnf copr enable mczernek/vale && sudo dnf install vale
+wget https://github.com/errata-ai/vale/releases/download/v2.30.0/vale_2.30.0_Linux_64-bit.tar.gz
+
+tar -xvzf vale_2.30.0_Linux_64-bit.tar.gz vale
+
+mv vale ${HOME}/bin/vale
+
+rm vale_2.30.0_Linux_64-bit.tar.gz
+
+echo "Installed $(vale --version) to ${HOME}/bin/vale"
```
Comment (kowen-rh) [inline modules/user-guide/pages/installing-vale-cli.adoc:50]: I don't think this is necessary for commands that are manually run by the reader.

File: modules/user-guide/pages/installing-vale-cli.adoc [hunk 2]
```diff
@@ -19,13 +19,35 @@ To use the various Vale at Red Hat styles, use Vale version 2.30.0 or lower.
...
 
-. To install the Vale command-line tool on RHEL or Fedora, run the following command:
+. To install the Vale command-line tool on RHEL or Fedora, run the following commands:
 +
 [source,terminal]
 ----
-$ sudo dnf copr enable mczernek/vale && sudo dnf install vale
+wget https://github.com/errata-ai/vale/releases/download/v2.30.0/vale_2.30.0_Linux_64-bit.tar.gz
+
+tar -xvzf vale_2.30.0_Linux_64-bit.tar.gz vale
```
[the remark this answers is not in the document]
Comment (aireilly) [reply modules/user-guide/pages/installing-vale-cli.adoc]: even better Thanks :)

File: modules/user-guide/pages/installing-vale-cli.adoc (hunk 1 above)
Comment (aireilly) [reply modules/user-guide/pages/installing-vale-cli.adoc:46]: `$(vale --version)` just to ensure that it got installed properly, and `${HOME}/bin/vale` in case they need to copy the location to an editor config :shrug:

Comment (kowen-rh) [reply modules/user-guide/pages/installing-vale-cli.adoc:46]: Fair enough. It's also easy to skip when running commands yourself. :)

Comment (rolfedh) [reply modules/user-guide/pages/installing-vale-cli.adoc:46]: I like having this verification step. Nice touch, IMO.

**Comment by aireilly** [comment]
Merging this for now since we need a valid install path for Fedora CSB. We can revist as soon as @daobrien has a chance to update the mczernek/vale RPM. Thanks!
